presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraDistributed.java

```diff
@@ -129,4 +129,10 @@ public void testWrittenStats()
     {
         // Cassandra connector currently does not support create table nor insert
     }
+
+    @Override
+    public void testShowTablesLikeWithEscape()
+    {
+        // Cassandra connector currently does not support create table, so we run empty test
```

Comment:
remove `, so we run empty test`

(i know i recommended adding `so we run simplified test`, but at that version of code there was _some_ test and documenting how it differs from superclass could be useful)